packages/destination-actions/src/destinations/podscribe/index.ts

```diff
@@ -0,0 +1,51 @@
+import type { Settings } from './generated-types'
+import { defaultValues, DestinationDefinition } from '@segment/actions-core'
+
+import track from './track'
+import page from './page'
+
+const presets: DestinationDefinition['presets'] = [
+  {
+    name: 'Order Completed Calls',
+    subscribe: 'type = "track" and event = "Order Completed"',
+    partnerAction: 'track',
+    mapping: {
+      ...defaultValues(track.fields),
+      podscribeEvent: 'purchase'
+    }
+  },
+  {
+    name: 'Signed Up Calls',
+    subscribe: 'type = "track" and event = "Signed Up"',
+    partnerAction: 'track',
+    mapping: {
+      ...defaultValues(track.fields),
+      podscribeEvent: 'signup'
+    }
+  }
```

Comment:
Consider adding a preset for page() events. 

```suggestion
  },
    {
    name: 'Page Calls',
    subscribe: 'type = "page"',
    partnerAction: 'page',
    mapping: defaultValues(page.fields)
  }
```